src/components/common/section-templates/single-title-description-author-pairs/SingleTitleDescriptionAuthorPairs.module.scss

```
@@ -68,24 +68,23 @@
 
-    :global(.char-limit-input) {
+    :global(.char-limit-textarea) {
         width: 721px;
-        height: 68px;
         box-shadow: none;
     }
 
-    :global(.char-limit-input__field) {
+    :global(.char-limit-textarea__field) {
         @include type.h3-bold;
         color: c.$blue-900;
```

Stylelint (17.7.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)